packages/app/src/sandbox/eval/transpilers/babel/worker/babel-worker.js

```diff
@@ -408,6 +408,21 @@ async function compile(code, customConfig, path, isV7) {
   try {
     let result;
     try {
+      // Set modern targets for env preset, speeds up Babel significantly
```

Comment:
When debugging a resolver issue I noticed the babel preset-env was running ie9 ? which is actually less modern than the codesandbox editor and causes a lot more transpilation and therefore slower builds

Reply:
I can always update the browserlist to something less modern but ie9 seems a bit extreme